src/backend/controllers/userController.js

```diff
@@ -1,24 +1,20 @@
 // @flow strict
 
 // Required imports
+import type { $Request,	$Response } from 'express';
```

Comment:
Not sure why it looks weird here, it looks normal when you go to edit it here and on my local machine

Reply:
you had a tab instead of a space